File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        
        //Approach 2 T.C. O(N) S.C.(1)
        
        int i=0;
        for(int j=1 ; j<nums.size(); j++)
        {
            if(nums[i] != nums[j]){
                //that means this is unique element form privious one
                nums[i+1] = nums[j];
                i++;
            }
        }
        return i+1;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        
        int current = 1;
        int counter = 0;
        for(int i=1; i<nums.length; i++){
            if(nums[i] > nums[i-1]){
                nums[current] = nums[i];
                current++;
                counter++;
            }
        }
        return counter+1;
    }
}